templates/bundles/TwigBundle/Exception/error403.html.twig

```diff
@@ -0,0 +1 @@
+{% include '@SyliusShop/error403.html.twig' %}
```

Comment:
We need to use it this way:
```suggestion
{% include ['@SyliusShop/error403.html.twig', @SyliusUi/error.html.twig] %}
```
Therefore, we will serve `error403` if it exists, and a generic error otherwise. Also, can you do the same for other errors in this folder?